src/lib/bindata/util.ts

```diff
@@ -28,3 +28,20 @@ export const uvarint64ToBuf = (uint: number): Buffer => {
 
   return new Buffer(result);
 };
+
+export const uint64ToBufBigEndian = (uint: number): Buffer => {
```

Comment:
DH: Where'd you get this from. Are you 100% sure it works?

Reply:
@diamondhands0 this is essentially doing the same thing as [`PutUint64`](https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/encoding/binary/binary.go;l=130) from the binary package that we use in core in [`EncodeUint64`](https://github.com/deso-protocol/core/blob/be25e74a93c93068cc4314c65fb26f99ddc70e52/lib/db_utils.go#L1838)  - and then we use this in [`_verifyAccessSignature()`](https://github.com/deso-protocol/core/blob/be25e74a93c93068cc4314c65fb26f99ddc70e52/lib/block_view.go#L7939 ). I wrote this myself, tested it, and it seems to work. Regardless, if it fails for some reason, this would only impact the UX. Core will reject the discrepancy in access signature so user will have to generate another derived key.